hazelcast-command-line/src/main/java/com/hazelcast/commandline/HazelcastCommandLine.java

```diff
@@ -0,0 +1,195 @@
+    }
+
+    private static PrintWriter createPrintWriter(PrintStream printStream) {
+        return new PrintWriter(new OutputStreamWriter(printStream, StandardCharsets.UTF_8));
+    }
+
+    @Override
+    public void run() {
+    }
+
+    @Command(description = "Starts a new Hazelcast member", mixinStandardHelpOptions = true, sortOptions = false)
+    void start(
+            @Option(names = {"-c", "--config"}, paramLabel = "<file>", description = "Use <file> for Hazelcast configuration. "
+                    + "Accepted formats are XML and YAML. ")
+                    String configFilePath,
+            @Option(names = {"-p", "--port"}, paramLabel = "<port>",
+                    description = "Bind to the specified <port>. Please note that if the specified port is in use, "
+                            + "it will auto-increment to the first free port. (default: 5701)")
+                    String port,
+            @Option(names = {"-i", "--interface"}, paramLabel = "<interface>",
+                    description = "Bind to the specified <interface>.")
+                    String hzInterface,
+            @Option(names = {"-j", "--jar"}, paramLabel = "<path>", split = ",", description = "Add <path> to Hazelcast "
+                    + "classpath (Use ',' to separate multiple paths). You can add jars, classes, "
+                    + "or the directories that contain classes/jars.")
+                    String[] additionalClassPath,
+            @Option(names = {"-J", "--JAVA_OPTS"}, paramLabel = "<option>", parameterConsumer = JavaOptionsConsumer.class,
+                    split = ",", description = "Specify additional Java <option> (Use ',' to separate multiple options).")
+                    List<String> javaOptions,
+            @Option(names = {"-d", "--daemon"},
+                    description = "Starts in daemon mode.")
+                    boolean daemon,
+            @Option(names = {"-v", "--verbose"},
+                    description = "Output with FINE level verbose logging.")
+                    boolean verbose,
+            @Option(names = {"-vv", "--vverbose"},
+                    description = "Output with FINEST level verbose logging.")
+                    boolean finestVerbose)
+            throws IOException, InterruptedException {
+        List<String> args = new ArrayList<>();
+        if (!isNullOrEmpty(configFilePath)) {
+            args.add("-Dhazelcast.config=" + configFilePath);
+        }
+        if (!isNullOrEmpty(port)) {
+            args.add("-Dnetwork.port=" + port);
+        }
+        if (!isNullOrEmpty(hzInterface)) {
+            args.add("-Dnetwork.interface=" + hzInterface);
+        }
+
+        args.add("-Djava.net.preferIPv4Stack=true");
+        addLogging(args, verbose, finestVerbose);
+
+        if (javaOptions != null && !javaOptions.isEmpty()) {
+            addJavaOptionsToArgs(javaOptions, args);
+        }
+
+        buildAndStartJavaProcess(args, additionalClassPath, daemon);
```

Comment:
Did you figure out why this starts a new process? Why can't we call the HazelcastMember main method directly?

Reply:
I think it was done to add additional jars to the classpath with `--jar` option and to setup JAVA_OPTS with `--JAVA_OPTS`

Reply:
It turned out that we need a new process to be able to run HZ in daemon mode. I've added `nohup` to be on the safe side.

Since we need a subprocess I'm wondering if we should keep `--jar` and `--JAVA_OPTS`. @frant-hartm any thoughts?